readme-template.md

```diff
@@ -1,106 +1,93 @@
 # Repo Title
 
-<!-- Not all of these are essential but keep this in mind for the repo you are using -->
-![Build badge]() ![Test Coverage]() ![npm package]()
+<!-- Not all of these are essential but keep this in mind for the repo you are using
+![Build badge]() ![Test Coverage]() ![npm package]() -->
 
 <!--Short sentence on what the repo does-->
 
 <!-- possibly list experts in this repo for who to go to when encountering a problem-->
 
 ##Contents
-* [Further Documentation](#documentation)
-* [System Overview](#overview)
+* [Further Documentation](#further-documentation)
+* [System Overview](#system-overview)
 * [Installation](#installation)
-* [Running Locally](#running)
+* [Running Locally](#running-locally)
 * [Testing](#testing)
 	* [Automated Testing](#automated)
 	* [Manual testing](#manual)
 * [Script cheat sheet](#cheat-sheet)
 * [Built With](#built-with)
-* [Deploying](#deploy)
 * [Linting](#linting)
-* [Contributing Rules](#rules)
-* [Common Questions](#faq)
+* [Deploying](#deploying)
+* [Contributing Rules](#contributing-rules)
+* [Common Questions](#common-questions)
 
-<a name="documentation"></a>
 ## Further Documentation
 
+<!-- Examples of some documentation to link from the readme -->
 * **[System Roadmap]()**
 * **[Regression Testing Checklist]()**
 * **[Change Log]()**
 
 Include any other relevant documentation in this section.
 
-<a name="overview"></a>
 ## System Overview
 
 * More in-depth overview of the system, what this is trying to accomplish and how it achieves its goal.
 * Keep it concise, write this section like the first paragraph of a blog post, what is the point of this repo?
+* Perhaps label some of the core tech used in the repo and link to the external websites/documentation
 
-<a name="installation"></a>
 ## Installation
 
-* Recordit of installation setup for new users
 * Step-by-step guide on installation
```

Comment:
Possibly add "run through this with someone unfamiliar with the repo and see if it's clear to understand" or something similar